### ISSUE TITLE
Documentation of language-specific features

### ISSUE THREAD

**Issue by jnivre** [UniversalDependencies/docs#151]
In the universal documentation, there are two sections that refer to language-specific categories:
http://universaldependencies.github.io/docs/ext-feat-index.html
http://universaldependencies.github.io/docs/ext-dep-index.html

These are rather different in nature. The first lists a large number of _potential_ language-specific features from Interset. The second lists the _actual_ language-specific relations used by different languages (although it is currently incomplete). I think these should be parallel and in both cases focus on the labels that are actually used. The list of potential features is clearly very useful, but I prefer to see it moved to the bottom of the page or even to a separate page. At the top of the page, I think there should be a list of language-specific features with links to the languages that use them, in a format similar to the one for language-specific relations. 

In addition, it would be great if the list of language-specific features (as well as that of language-specific relations) could be generated automatically from the language-specific documentation pages. This would not only save work but would also ensure consistency.

**Comment by dan-zeman** [comment]
Agree.
The primary purpose of listing the extra features of Interset is to inspire people who would think of encoding something that is already in Interset, and thus to get language-specific-but-compatible implementation of these features.
The features that were actually used in documented languages should certainly be shown first, and in a similar fashion to the page with dependency relations.

**Comment by dan-zeman** [comment]
I have restructured the page on language-specific features. (c5277b0f5114ea0403c195b2d8443d299ff63554). The appearance is now closer to the page on language-specific relations and there are links for particular languages (I did not search in all the languages and I only found relevant stuff in Czech and Finnish. But this is enough to show how people should add links to their languages.) I also decided to keep it mixed with the links to Interset features that are not yet defined in any language covered by UD. The point is that I expect these features to be needed once the respective languages get covered, so I want the owners of those languages to see the Interset solution in the current feature menu, and to consider using it.

Tentatively closing the issue.
